### ISSUE TITLE
NumberBox keyboard navigation requires 2 tabs

### ISSUE THREAD

**Issue by jesbis** [microsoft/microsoft-ui-xaml#1706]
**Describe the bug**
It takes two tab keystrokes to move focus in/out of a NumberBox, and in between the key events seem to bubble up to the parent. 

**Steps to reproduce the bug**

1. Add NumberBox to a page
2. Attempt to tab and shift-tab back and forth in and out of a NumberBox

**Expected behavior**
Keyboard focus navigation should be consistent with other controls: it should only take 1 tab to get in/out of a NumberBox, similar to TextBox etc.

**Version Info**

NuGet package version: 
Microsoft.UI.Xaml 2.3.191129002

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2019 Update (18362)            | Yes   |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes  |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**
